crates/binjs_io/src/entropy/dictionary.rs

```diff
@@ -250,14 +249,18 @@ where
     /// If the value is already in the cache, return `Fetch::Hit(index)`, where `index` is the
     /// immutable index of the value. Otherwise, allocate a new slot `index` and return
     /// `Fetch::Miss(index)`.
-    pub fn fetch_index(&mut self, value: &T) -> Fetch {
+    ///
+    /// In both `Fetch::Hit(index)` and `Fetch::Miss(index)`, `0` is the first value, `1` the second
+    /// value, etc. In particular, you received `Fetch::Hit(i)` and `Fetch::Miss(j)`, `i` MAY BE
+    /// equal to `j`.
```

Comment:
I'm still not sure what this sentence actually means.
does it mean that second fetch to the same thing could be `Hit` while the first fetch returned `Miss`,
or something else?

Reply:
It means that we can have a call that returns `Fetch::Hit(0)` and another one that returns `Fetch::Miss(0)`.

Reply:
what's the situation?
how is it expected or unexpected?

Reply:
I'll just remove this comment. It's not really useful.